rock_scissor_paper_assignment.js

```diff
@@ -1 +1,29 @@
-// code here
+document.getElementById("play").addEventListener("click", btnClick);
+
+function btnClick() {
+  var player1 = document.getElementById('player-choice').value;
+  document.getElementById("player-result").innerHTML = player1;
+  var player2 = Math.floor(Math.random() * 3);
+  document.getElementById("computer-result").innerHTML = player2;
+
+  document.getElementById('pic1').innerHTML = '<img src='+ checkPics(player1) + '>';
```

Comment:
👏 this is nice! small advice, you can create image object which represents the image element on the DOM and then set attributes

http://www.w3schools.com/jsref/dom_obj_image.asp